apps/storybook/package.json

```diff
@@ -24,6 +24,7 @@
     "@h5web/lib": "workspace:*",
     "@h5web/shared": "workspace:*",
     "@react-hookz/web": "14.2.2",
+    "@react-three/drei": "8.20.2",
```

Comment:
Added `drei` to the storybook for the [OrbitControls](https://drei.pmnd.rs/?path=/story/controls-orbitcontrols--orbit-controls-story).

I think this is the best compromise for now.